bindings/pyfiction/include/pyfiction/utils/version_info.hpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by Jan Drewniok on 11.11.24.
+//
+
+#ifndef PYFICTION_VERSION_INFO_HPP
+#define PYFICTION_VERSION_INFO_HPP
+
+#include <fiction/utils/version_info.hpp.in>
+
+#include <pybind11/pybind11.h>
+
+namespace pyfiction
+{
+
+using namespace pybind11::literals;
+
+inline void version_info(pybind11::module& m)
+{
+    m.attr("FICTION_VERSION") = fiction::FICTION_VERSION;
+    m.attr("FICTION_REPO")    = fiction::FICTION_REPO;
+    m.attr("COMPILED_DATE")   = fiction::COMPILED_DATE;
+    m.attr("COMPILED_TIME")   = fiction::COMPILED_TIME;
```

Comment:
I believe Python has a standardized form for these variables like `__version__`, `__url__`, etc. Could you look this up and set the names accordingly?

Reply:
oh, you are right! That's a good point.